src/docs/product/data-management-settings/dynamic-sampling/operations.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Dynamic Sampling Operations
+sidebar_order: 30
+description: "Learn more about how dynamic sampling rules are applied."
+---
+
+Dynamic sampling operates on both error events and transaction events.
+
+## Error Sampling Decisions
+
+Error sampling decisions are made based on the event data.
+
+Sentry selects a sampling rule from those available by going through the list of dynamic sampling rules for errors, then picking the first rule that matches the event data. Rule matching is described in more detail below.
+
+## Transactions Sampling Decisions
+
+_ **TODO** This section seems to have quite a bit of overlap with the Transactions section in Sampling Types, decide what to remove._
```

Comment:
Recommend keeping this content here and removing the Dynamic Sampling Operations section (with sub-sections Errors and Transactions) from the "Sampling Types" page since the content doesn't really seem to fit the subject matter of that page.